doc/JSON Mapping Guides/Guide for intermediate mapgen.md

```diff
@@ -0,0 +1,614 @@
+        " F $$$$$$$,,,,$$$$$$$ f  7   7 ,,7 ?  7   7   7  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f?     ? ,,          ?     f                    ,,,,                    F ",
+        " F        ,,,,        f  ,,,,,,,,,,,,,,,,,,,,,,  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F $$$$$$$,,,,$$$$$$$ f        ,,       ?        f   ,           ,    ,,,,  ,          ,      F ",
+        " F        ,,,,        f  7   7 ,,7  ? 7   7   7  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f        ,,              ? f                 ,  ,,,,                    F ",
+        " F $$$$$$$,,,,$$$$$$$ f  ,,,,,,,,,,,,,,,,,,,,,,  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f    ?   ,,  ?             f,                   ,,,,    ,               F ",
+        " F        ,,,,        f  7   7 ,,7    7   7   7  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F $$$$$$$,,,,$$$$$$$ f        ,,       ?        f   ,      ,         ,,,,  ,         ,       F ",
+        " F        ,,,,        f  ,,,,,,,,,,,,,,,,,,,,,,  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f ?      ,,           ?    f        Q           ,,,,                    F ",
+        " F $$$$$$$,,,,$$$$$$$ f  7   7 ,,7    7   7   7  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f        ,,,  ?           ?f                ,   ,,,,          ,        ,F ",
+        " F        ,,,,     Q  f  ,,,,,,,,,,,,,,,,,,,,,,  f DDDDDDDDDDDDDDDDDDD,,,,DDDDDDDDDDDDDDDDDDD F ",
+        " F $$$$$$$,,,,$$$$$$$ f    ?     ,,,         ?   f     ,       ,      ,,,,                    F ",
+        " F        ,,,,        f  7   7   7,,  7   7   7  f,DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f       ?  ,,,     ?       f                 ,     ,        ,     Q     F ",
+        " F $$$$$$$,,,,$$$$$$$ f  ,,,,,,,,,,,,,,,,,,,,,,  f DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD F ",
+        " F        ,,,,        f  ?       ,,              f             ,              ,           ,   F ",
+        " F                    f  7   7   7  ? 7   7   7  f DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD F ",
+        " IFFFFFFFFFFFFFFFFFFFFIFFFFFFFFFFFFFFFFFFFFFFFFFFIFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFI "
+      ],
+```
+
+Important note about spawning items, vehicles and monsters:
+
+While these maps are merged for the benefit of readability, this isn't automatically recognized by our mapgen processes.  When you add spawns using x,y coordinates you can't cross OMT boundaries.  Make multiple entries if you want those spawns to occur across more than one OMT.  This applies for both x and y coordinates.
+
+An example:  each x coordinate encompasses one OMT from a segment of the mall.
+```
+"place_monsters": [
+        { "monster": "GROUP_MALL", "x": [ 2, 23 ], "y": [ 2, 23 ], "density": 0.15 },
```

Comment:
Would be nice to have a few words regarding how `density` works. I wouldn't say it's really intuitive.

Reply:
Thats more a monster.md topic but I'll point to there.  I also don't understand it well enough to document it.